### PR TITLE
https://issues.redhat.com/browse/ACM-12758 Link related sections

### PR DESCRIPTION
**Pull request by oafischer**
2.12 only

### CODE REVIEW

**Review by showeimer — APPROVED** [review]
lgtm

File: clusters/cluster_lifecycle/adv_config_cluster.adoc [hunk 1]
```diff
@@ -298,7 +300,8 @@ spec:
 
 If you create another `KlusterletConfig` in an environment that has a global `KlusterletConfig` and bind it with a managed cluster, the value of the global `KlusterletConfig` is used if the same field is missing or does not have a value in your `KlusterletConfig`.
 
-See the following examples, where the `"example.global.com"` value in the `hubKubeAPIServerURL` field of the global `KlusterletConfig` overrides your `KlusterletConfig`.
+See the following examples, where the `"example.global.com"` value in the `hubKubeAPIServerURL` field of the global `KlusterletConfig` overrides your `KlusterletConfig`:
+//these two examples need some kind of header or intro because a nonsighted user does not have any idea what is coming after the colon. Example one displays... Example two displays.
```
Comment (swopebe) [inline clusters/cluster_lifecycle/adv_config_cluster.adoc]: @oliver I scanned the file and noticed we don't alert the user with a screen reader what to look for here. We needed a colon, so I added that, but can you look at the format for these examples?

File: clusters/cluster_lifecycle/adv_config_cluster.adoc [hunk 2]
```diff
@@ -339,6 +342,7 @@ spec:
 * link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.14/html/security_and_compliance/configuring-certificates#api-server-certificates[Adding API server certificates]
 
 //need to change to link: keeping the comment to test Oliver's theory later|MJ|07/23
+//can we check on this and remove comment?
```
Comment (swopebe) [inline clusters/cluster_lifecycle/adv_config_cluster.adoc]: I am on a mission to clean up comments before we have major changes to the repo....do you mind checking on this one?

**Review by swopebe — APPROVED** [review]
Approving but please see the two built in comments and remove when ready, one is for accessibiity.

File: clusters/cluster_lifecycle/adv_config_cluster.adoc (hunk 2 above)
Comment (oafischer) [reply clusters/cluster_lifecycle/adv_config_cluster.adoc]: Removed comments

File: clusters/cluster_lifecycle/adv_config_cluster.adoc (hunk 1 above)
Comment (oafischer) [reply clusters/cluster_lifecycle/adv_config_cluster.adoc]: Updated into separate examples

**Pull request merged**